utils/filesystem.go

```diff
@@ -72,17 +73,16 @@ func WriteToComposeFile(tempFilePath string, debug bool) bool {
 }
 
 // DeleteTempFile once the the Codewind environment has been created
-func DeleteTempFile(tempFilePath string) (boolean bool, err error) {
-
-	var _, file = os.Stat(tempFilePath)
+func DeleteTempFile(filePath string) (bool, error) {
+	var _, file = os.Stat(filePath)
 
 	if os.IsNotExist(file) {
 		errors.CheckErr(file, 206, "No files to delete")
 		return false, file
 	}
 
-	os.Remove(tempFilePath)
-	fmt.Println("==> finished deleting file " + tempFilePath)
+	os.Remove(filePath)
+	log.Printf("Deleted file: %s\n\n", filePath)
```

Comment:
What is the reason for changing this to `log`? By default `log` prints to stderr. I dont think this needs to be in stderr and `fmt` is sufficient.